app/javascript/src/shared/styles/fontfaces.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import { css } from 'glamor'
+import robotoLightWOFF2 from '../../../fonts/roboto-light.woff2'
```

Comment:
can we camel-case these names, e.g. `robotoLightWoff2`? also, was there a problem with exporting all of these fonts from a shared index to clean up this import? i can't remember.

Reply:
Aaah yes you're right. We set that up and then while I was debugging something else I removed that. Will switch back, and fix camel-case.